components/link/stories/template.js

```diff
@@ -43,9 +45,52 @@ export const Template = ({
 };
 
 export const TemplateWithFillerText = (args, context) => html`
-	<div>
+	<p>
 		Hello, this is a
 		${Template(args, context)}
 		. This is just filler text, but if you keep reading maybe something good will happen.
-	</div>
+	</p>
 `;
+
+export const MultilineText = (args, context) => html`
+	<p style="width: 200px;">
+		I like focus styles. They are very important for accessibility. They help users know where they are on the page.
+		${Template({
+			...args,
+			text: "This is a link that spans multiple lines",
+			isFocused: true,
+			isInline: true,
+		}, context)}
+		They are also very pretty.
+	</p>
+`;
+
+export const LinkGroupText = (args, context) => Container({
+	withBorder: false,
+	direction: "row",
+	content: html`
+	${Container({
+		withBorder: false,
+		direction: "column",
+		heading: "Default",
+		content: html`
+			${Template({
+				...args,
+				text: "Learn more",
+			}, context)}
+		`,
+	}, context)}
+	${Container({
+		withBorder: false,
+		direction: "column",
+		heading: "Quiet",
+		content: html`
+			${Template({
+				...args,
+				text: "Learn more",
+				isQuiet: true,
+			}, context)}
+		`,
```

Comment:
```suggestion
		content: Template({
			...args,
			text: "Learn more",
			isQuiet: true,
		}, context)
```

This can simplify it a bit for you since the Template returns an html object already - you don't have to wrap it again if you don't want to.